src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Sidebar from './Sidebar';
+
+export default function App() {
+  return (
+    <Sidebar>
+      <p>Home</p>
```

Comment:
These could be wrapped in <a> to create links, that when clicked, can update state! Think of how router works now that you have that tool.